components/layout/BluerpintsCardsCarusel.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+    <div class="container">
+        <div class="title mb-3">
+            <p>Related Blueprints</p>
+        </div>
+        <div class="mt-3">
+            <Carousel v-bind="settings" :breakpoints="breakpoints">
+                <Slide v-for="blueprint in blueprints" :key="blueprint.id" >
+                    <div class="carousel--item">
+                        <BlueprintsListCard :blueprint="blueprint" :tags="tags"/>
+                    </div>
+                </Slide>
+                <template #addons>
+                    <navigation>
+                        <template #next>
+                            <div class="carousel-control carousel-control-next">
+                                <ChevronRight />
+                            </div>
+                        </template>
+                        <template #prev>
+                            <div class="carousel-control carousel-control-prev">
+                                <ChevronLeft />
+                            </div>
+                        </template>
+                    </navigation>
+                </template>
+            </Carousel>
+        </div>
+    </div>
+</template>
+<script setup>
+  const config = useRuntimeConfig();
+  const blueprints = ref([])
+
+  const { data: blueprintsData } = await useAsyncData('blueprints', () => {
+    return $fetch(`${config.public.apiUrl}/blueprints`)
```

Comment:
You need to pass the task type here as query string `types`

Reply:
@tchiotludo can you show me how should look query ?

Reply:
`${config.public.apiUrl}/blueprints?type=${cls}`
with cls = "io.kestra.plugin.jdbc.pinot.trigger" for examples